modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/notification-sidebar/index.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import classNames from 'classnames';
+import {useEffect, useState} from 'react';
+
+import './index.scss';
+import {
+	getUserNotification,
+	putUserNotificationRead,
+} from '../../../common/services/Notification';
+import {PostType} from './postTypes';
+
+const initialItems = {page: 1, pageSize: 4, totalCount: 0};
+
+const NotificationSidebar: React.FC = () => {
+	const [posts, setPosts] = useState<PostType[]>([]);
+	const [totalCount, setTotalCount] = useState<number>(
+		initialItems.totalCount
+	);
+	const [linkUrl, setLinkUrl] = useState<string>();
+	const [page, setPage] = useState<number>(initialItems.page);
+
+	useEffect(() => {
+		getNotifications();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [page]);
+
+	async function getNotifications() {
+		const response = await getUserNotification(initialItems.pageSize, page);
+
+		const notifications = response?.data;
+
+		const notificationsTotalCount = notifications?.totalCount;
+
+		setTotalCount(notificationsTotalCount);
```

Comment:
```suggestion
		const notificationsTotalCount = notifications?.totalCount;

		setTotalCount(notifications?.totalCount);
```
Do you need this variable for other thing? Because I believe you can just set the state here. What do you think?